scss/_base_code.scss

```diff
@@ -0,0 +1,28 @@
+// Base code styles
+@mixin vf-b-code {
+  code,
+  samp,
+  kbd {
+    font-family: $font-monospace;
+    text-align: left;
+  }
+
+  pre,
+  code {
+    text-align: left;
+    white-space: pre-line;
```

Comment:
`pre-line` stripes spaces and tabs from the code blocks. If QA this branch and toggle this style you will see that the code goes from all aligned left to containing the spaces and tabs included in the markup of the page.